client/components/token-field/suggestions-list.jsx

```diff
@@ -86,13 +91,42 @@ class SuggestionsList extends React.PureComponent {
 		);
 	}
 
+	_removeEmptyLabelsFromSuggestions( suggestions ) {
+		const filteredSuggestions = [];
+		for ( let i = 0; i < suggestions.length; i++ ) {
+			if (
+				isSuggestionLabel( suggestions[ i ] ) &&
+				( ( suggestions?.[ i + 1 ] && isSuggestionLabel( suggestions[ i + 1 ] ) ) ||
+					! suggestions?.[ i + 1 ] )
```

Comment:
I believe this can be simplified to
```
isSuggestionLabel( suggestions[ i ] ) &&
	( ! suggestions?.[ i + 1 ] || isSuggestionLabel( suggestions[ i + 1 ] ) );
```